src/test/java/reposense/commits/CommitInfoAnalyzerTest.java

```diff
@@ -125,6 +125,26 @@ public void analyzeCommits_emailWithAdditionOperator_success() {
         Assert.assertEquals(NUMBER_MINGYI_COMMIT, commitResults.size());
     }
 
+    @Test
+    public void analyzeCommits_duplicateAuthorsDuplicateCommits_success() throws ParseException {
+        Author author = new Author(EUGENE_AUTHOR_NAME);
+        List<CommitResult> expectedCommitResults = new ArrayList<>();
+        expectedCommitResults.add(new CommitResult(author, LATEST_COMMIT_HASH,
+                parseGitStrictIsoDate("2018-11-08T13:50:40+08:00"),
+                "Add config.json with invalid fields (#2)",
+                "", null, 17, 0));
+
+        config.setAuthorList(Arrays.asList(author, author));
```

Comment:
The test does achieve the goal of verifying that duplicate commits are removed.
However, when we create the author list, would it be better if one of the author instances has the author name `eugenepeh` and the other instance has the author name `Eugene Peh` ?

I understand that the bug can be replicated with the current test setup too, but perhaps using `eugenepeh` and `Eugene Peh` allows us to better simulate the bug that appears in practice?

Reply:
Hmm, there's the system test for that already though, I feel that it may be better to keep the duplicate authors here so the purpose of the test `duplicate commits are removed` is clearer.
Otherwise we are also testing the commit tagging behaviour here ( which should be fixed )

Reply:
Ok - yes the system test does cover it already